content/manual-references.json

```diff
@@ -942,4 +942,26 @@
     "type": "webpage",
     "standard_citation": "url:https://nihcc.app.box.com/v/ChestXray-NIHCC"
   }
+{
+  "standard_citation": "doi:10.3389/fgene.2014.00342",
+  "type": "research-article",
+  "title": "Large-scale integration of small-molecule induced genome-wide transcriptional responses, Kinome-wide binding affinities and cell-growth inhibition profiles reveal global trends characterizing systems-level drug action",
+  "source": "www.frontiersin.org",
+  "URL": "https://www.frontiersin.org/articles/10.3389/fgene.2014.00342/full",
```

Comment:
@dhimmel should we leave out the URL?  For typical DOI citations the URL would be constructed from the DOI, but I'm not positive that is true for manual references.

Reply:
I'd keep the URL but change it to the DOI URL: https://doi.org/10.3389/fgene.2014.00342